src/lib/thanos/back/dapp.ts

```diff
@@ -29,6 +31,7 @@ import { loadChainId, isAddressValid } from "lib/thanos/helpers";
 const CONFIRM_WINDOW_WIDTH = 380;
 const CONFIRM_WINDOW_HEIGHT = 600;
 const AUTODECLINE_AFTER = 120_000;
+const STORAGE_KEY = "dApps";
```

Comment:
Maybe go `"dapp_sessions"`:
1) In lower-case like other storage keys;
2) with "sessions" postfix because in the future there may be other keys related to dApps.

Reply:
Actually, why not?